patches/tModLoader/Terraria/Item.cs.patch

```diff
@@ -1463,6 +1463,22 @@
  	public void DefaultToSpear(int projType, float pushForwardSpeed, int animationTime)
  	{
  		useStyle = 5;
+@@ -46845,6 +_,15 @@
+ 		rare = (int)(num / 4f);
+ 	}
+ 
++	/// <summary>
++	/// This method sets a variety of Item values common to mount items.<br/>
++	/// Specifically: <code>
++	/// width = 36;
++	/// height = 26;
++	/// mountType = mount;
++	/// </code>
++	/// </summary>
++	/// <param name="mount">The mount type</param>
+ 	public void DefaultToMount(int mount)
```

Comment:
Not sure this should be public, or should be called `DefaultToMinecart`, should we refactor+tModPorter?

Reply:
Same as https://github.com/tModLoader/tModLoader/pull/3477#discussion_r1250996981